src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Flex, Sans, Separator, Serif } from "@artsy/palette"
+import React from "react"
+
+export const VanguardFooter = () => {
+  return (
+    <Box background="white" pb={80}>
+      <Box
+        maxWidth={["100vw", 1000]}
+        px={["10vw", "10vw", "10vw", "10vw", 4]}
```

Comment:
#minor — I think styled-system allows us to pass `null` for unchanging values, instead of repeating them. This should also result in a (small) optimization of the generated css, since redundant breakpoints wouldn't be written out.